dir.traversal.targets

```diff
@@ -216,9 +215,43 @@
     <Error Condition="'$(MSBuildLastTaskResult)'=='false'" />
   </Target>
 
+ <Target Name="AnnotateProjectsWithConfiguration" 
```

Comment:
Is your assumption that this only happens for Project's which are themselve's traversals? Wouldn't we need this target for just plan projects as well?